dev_guide/builds.adoc

```diff
@@ -1515,6 +1515,11 @@ Therefore, if you are using a Gogs server, you can define a GitHub webhook
 trigger on your `*BuildConfig*` and trigger it via your Gogs server also.
 ====
 
+Given a file containing a valid json payload, you can manually trigger the webhook via `*curl*`:
+
+    $ curl -H "X-GitHub-Event: push" -H "Content-Type: application/json" -k -X POST --data-binary @github_payload_file.json https://<openshift_api_host:port>/oapi/v1/namespaces/<namespace>/buildconfigs/<name>/webhooks/<secret>/github
```

Comment:
Add ---- before and after the code snippet
(Per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)

So: 
----
$ curl -H "X-GitHub-Event: push" -H "Content-Type: application/json" -k -X POST --data-binary @github_payload_file.json https://<openshift_api_host:port>/oapi/v1/namespaces/<namespace>/buildconfigs/<name>/webhooks/<secret>/github
----